connector_importer/components/odoorecord.py

```diff
@@ -38,14 +40,43 @@ def unique_key_is_xmlid(self):
 
     def odoo_find_domain(self, values, orig_values):
         """Domain to find the record in odoo."""
+        domain = self._odoo_find_domain_from_options(values, orig_values)
+        if not domain:
+            domain = self._odoo_find_domain_from_unique_key(values, orig_values)
+        return domain
+
+    def _odoo_find_domain_from_options(self, values, orig_values):
+        """Evaluate domain from options if any."""
+        match_domain = self.work.options.record_handler.match_domain
+        if not match_domain:
+            return []
```

Comment:
I may be missing context here, but `[]` is a valid domain that one can configure through options, right?
If so this would be ignored in line 44, no?

Reply:
Hmm no, it's not. The domain should match only 1 record.